cmd/check-gh-automation/main.go

```diff
@@ -119,20 +121,26 @@ func checkRepos(repos []string, bots []string, ignore sets.String, client collab
 
 		var missingBots []string
 		for _, bot := range bots {
-			isCollaborator, err := client.IsCollaborator(org, repo, bot)
+			isMember, err := client.IsMember(org, bot)
 			if err != nil {
-				return nil, fmt.Errorf("unable to determine if: %s is a collaborator on %s/%s: %v", bot, org, repo, err)
+				return nil, fmt.Errorf("unable to determine if: %s is a member of %s: %v", bot, org, err)
 			}
-			if !isCollaborator {
-				missingBots = append(missingBots, bot)
+			if !isMember {
```

Comment:
Flatten the logic a bit?

if isMember {
  logger.withField("org", org).withField("repo", repo).withField("bot", bot).Info("bot is a member")
  continue 
}